.scenarios.lock/drupal9/composer.json

```diff
@@ -18,26 +18,33 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "drupal/core-recommended": "^9@rc",
+        "composer/installers": "^1.2",
         "drupal/core": "^9@rc",
         "drupal/core-composer-scaffold": "^9@rc",
-        "pantheon-systems/drupal-integrations": "^9",
-        "composer/installers": "^1.2",
-        "drush/drush": "^8.3.3 || ^10.2.2",
-        "g1a/composer-test-scenarios": "^3.0.2",
-        "phpunit/phpunit": "^6.5",
+        "drupal/core-recommended": "^9@rc",
         "drupal/php": "^1",
+        "drush/drush": "^10.4.0 || ^11.x-dev",
```

Comment:
Maybe this should be `"drush/drush": "10.4.0 || ^11.x-dev",` (remove the `^` in front of Drush 10)

Reply:
Or just take "10" out as an option, as it looks liek there aren't any 7.3 tests at the moment -- which I think is fine.

Reply:
Did you hand-edit the scenario lock files? I don't see the scenario definition for Drush 10.4.0.

Reply:
I didn't hand-edit lock files but I'm using "lowest" for 7.3 tests so that it installs the lowest possible version (which in this case should be 10.4.0)

Reply:
I'm trying to update the scenarios and run `composer update`, but running into problems because drupal-composer/drupal-scaffold isn't compatible with Composer 2. Seeing if I can remove that tool.

Reply:
I tried to fix it, but made it worse.

Reply:
Composer Test Scenarios is supposed to maintain both the composer.json and the composer.lock files in the .scenario directories, but it's not possible to generate the PHP 5.6 composer.lock file with Composer 2, so I removed the PHP 5.6 scenario from the root-level composer.json file, and left the others managed by Composer Test Scenarios.

My changes have introduced some failures, though.